.pre-commit-config.yaml

```diff
@@ -0,0 +1,26 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.0.1
+    hooks:
+      - id: check-docstring-first
+      - id: check-toml
+      - id: check-yaml
+        exclude: packaging/.*
+      - id: mixed-line-ending
+        args: [--fix=lf]
+      - id: end-of-file-fixer
+
+
+  - repo: https://github.com/omnilib/ufmt
+    rev: v1.3.0
+    hooks:
+      - id: ufmt
+        additional_dependencies:
+          - black == 21.9b0
+          - usort == 0.6.4
+
+
+  -   repo: https://github.com/pre-commit/mirrors-clang-format
+      rev: v13.0.0
```

Comment:
Is this clang compatible with the one PyTorch is using?

PyTorch core uses specific binaries so that the result is consistent across platforms.

https://github.com/pytorch/pytorch/blob/24b60b2cbf37f46818624f36df2e539f5b187a88/tools/linter/adapters/s3_init_config.json#L2-L15

Reply:
well, as long as it's consistent within torchtext, then there should not be an issue